services/shhext/api.go

```diff
@@ -427,13 +421,11 @@ func (api *PublicAPI) GetNewFilterMessages(filterID string) ([]dedup.Deduplicate
 
 	dedupMessages := api.service.deduplicator.Deduplicate(msgs)
 
-	if api.service.pfsEnabled {
-		// Attempt to decrypt message, otherwise leave unchanged
-		for _, dedupMessage := range dedupMessages {
+	// Attempt to decrypt message, otherwise leave unchanged
+	for _, dedupMessage := range dedupMessages {
```

Comment:
we will most likely get rid of deduplication service when moving messaging to status-go side... jfyi.